lib/survey_gizmo/api/survey.rb

```diff
@@ -20,25 +22,35 @@ class Survey
     attribute :created_on,     DateTime
     attribute :modified_on,    DateTime
     attribute :copy,           Boolean
+    # Unfortunately if pages is an attribute, then save and update requests will collide with the differing response
+    # types to Survey.all and Survey.first and cause an incorrect reload
+    # attribute :pages,          Array[Page]
```

Comment:
OK so this is the workaround - just don't use Virtus to load `Survey#pages`, because pages aren't returned to one of the two types of GET requests for surveys.

for `Pages#questions` and `Question#options`, Virtus autoload should work well, but if any other users run into errors I will just revert all this stuff to the extremely chatty way of getting object data out of surveygizmo because it's not really worth the debug time to deal with their inconsistency.